### DISCUSSION TITLE
Difference between two WSIReader

### DISCUSSION THREAD

**Discussion by KumoLiu** [Project-MONAI/MONAI#4776] [Q&A] [answered]
**Description**
I just find two `WSIReader` which one is in monai/data/wsireader.py and the other is in monai/data/image_reader.py. I would like to ask if there is any difference between these two files? Because the same name just confuse me. @drbeh Could you please help me figure out? 
https://github.com/Project-MONAI/MONAI/blob/31c4209f1d84a95536b56c3e4d6f7b5a31fe9b6f/monai/data/image_reader.py#L1207

https://github.com/Project-MONAI/MONAI/blob/31c4209f1d84a95536b56c3e4d6f7b5a31fe9b6f/monai/data/wsi_reader.py#L257
Thanks in advance!

**Comment by bhashemian** [comment, marked as answer]
Hi @KumoLiu,

The one in `image_reader` is going to be deprecated. Please use `WSIReader` in `wsi_reader.py`.

Best,
Behrooz

**Reply by Nic-Ma** [reply]
Hi @drbeh ,

So I think we can deprecate it now? Could you please help add the deprecated decorator for it? Maybe in PR https://github.com/Project-MONAI/MONAI/pull/4768?

Thanks.

**Reply by bhashemian** [reply]
Hi @Nic-Ma, I will do it in the pending PR that we already have for deprecating pathology components: #4415
